src/javascript/app/pages/user/account/authenticate.js

```diff
@@ -13,16 +13,27 @@ const Authenticate = (() => {
             if (response.error) {
                 $('#error_message').setVisibility(1).text(response.error.message);
             } else {
-                const status = response.get_account_status.status;
-                if (!/authenticated/.test(status)) {
-                    init();
-                    const $not_authenticated = $('#not_authenticated').setVisibility(1);
-                    let link = 'https://marketing.binary.com/authentication/2017_Authentication_Process.pdf';
-                    if (Client.isAccountOfType('financial')) {
-                        $('#not_authenticated_financial').setVisibility(1);
-                        link = 'https://marketing.binary.com/authentication/2017_MF_Authentication_Process.pdf';
+                const get_account_status  = response.get_account_status;
+                const should_authenticate = +get_account_status.prompt_client_to_authenticate;
+                const status = get_account_status.status;
+                if (should_authenticate) {
+                    if (!/authenticated/.test(status)) {
+                        init();
+                        $('#not_authenticated').setVisibility(1);
+                        // Show upload instructions
+                        if (!jpClient()) {
+                            const $not_authenticated = $('#not_authenticated').setVisibility(1);
+                            let link = 'https://marketing.binary.com/authentication/2017_Authentication_Process.pdf';
+                            if (Client.isAccountOfType('financial')) {
+                                $('#not_authenticated_financial').setVisibility(1);
+                                link = 'https://marketing.binary.com/authentication/2017_MF_Authentication_Process.pdf';
+                            }
+                            $not_authenticated.find('.learn_more a').attr('href', link);
+                        }
+
+                    } else if (!/age_verification/.test(status)) {
+                        $('#needs_age_verification').setVisibility(1);
```

Comment:
To fix according to the previous comment, will also fix the duplication here with the next lines.